### PR TITLE
AAE-2646 update image tag to develop after release so automation can …

### PR DESCRIPTION
**Pull request by ffazzini**
…point to latest code

### CODE REVIEW

File: helm/alfresco-process-infrastructure/values.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ alfresco-modeling-service:
       readOnly: true
   image:
     repository: quay.io/alfresco/alfresco-modeling-service
-    tag: 7.1.0-M7
+    tag: "develop"
```
Comment (mteodori) [inline helm/alfresco-process-infrastructure/values.yaml:122]: no need for quotes

File: helm/alfresco-process-infrastructure/values.yaml [hunk 2]
```diff
@@ -159,7 +159,7 @@ alfresco-modeling-app:
     path: /modeling
   image:
     repository: quay.io/alfresco/alfresco-modeling-app
-    tag: 7.1.0-M7
+    tag: "development"
```
Comment (mteodori) [inline helm/alfresco-process-infrastructure/values.yaml:162]: tag is now develop everywhere including front-end

File: helm/alfresco-process-infrastructure/values.yaml (hunk 1 above)
Comment (ffazzini) [reply helm/alfresco-process-infrastructure/values.yaml:122]: makes sense i got that from the history, changing it now

File: helm/alfresco-process-infrastructure/values.yaml (hunk 2 above)
Comment (ffazzini) [reply helm/alfresco-process-infrastructure/values.yaml:162]: perfect, changing it now

**Pull request merged**